### PR TITLE
[PLAY-753] fixed. Video does not play

### PR DESCRIPTION
**Pull request by RobertoUa**
@shamansir @skavish

### CODE REVIEW

File: src/engine/dom-engine.js [hunk 1]
```diff
@@ -924,7 +924,9 @@ $DE.createAudio = function() {
 };
 
 $DE.createVideo = function() {
-    return $doc.createElement('video');
+    var el = $doc.createElement('video');
+    el.width = el.height = "1";
```
Comment (shamansir) [inline src/engine/dom-engine.js]: shouldn't it be `1px` at least?

**Comment by RobertoUa** [comment]
updated

File: src/engine/dom-engine.js (hunk 1 above)
Comment (RobertoUa) [inline src/engine/dom-engine.js]: no

**Comment by shamansir** [comment]
ok

**Comment by RobertoUa** [comment]
updated for videos larger than 720

**Comment by shamansir** [comment]
@skavish do you approve?

File: src/anm/media/video.js [hunk 2]
```diff
@@ -137,7 +137,7 @@ Video.prototype.load = function(elm, player) {
 };
 /** @private @method apply */
 Video.prototype.apply = function(ctx) {
-    if (this.video) ctx.drawImage(this.video, 0, 0, this.video.width, this.video.height, 0, 0, this.size[0], this.size[1]);
+    if (this.video) ctx.drawImage(this.video, 0, 0, this.video.videoWidth, this.video.videoHeight, 0, 0, this.size[0], this.size[1]);
```
Comment (skavish) [inline src/anm/media/video.js]: yep, that's correct fix, at least those are always be set if the video is loaded.

I was contemplating what to use in the editor: width or videoWidth and for whatever reason chose width, although videoWidth is better

**Pull request merged**